annotationProcessor/src/main/java/org/corfudb/annotations/ObjectAnnotationProcessor.java

```diff
@@ -562,6 +593,47 @@ private void addUpcallMap(TypeSpec.Builder typeSpecBuilder, TypeName originalNam
                 .build());
     }
 
+    private void addConflictExtractorMap(TypeSpec.Builder typeSpecBuilder, TypeName originalName,
+                                         Set<SMRMethodInfo> methodSet) {
+        // Generate the upcall string and associated map.
+        String conflictExtractorString = methodSet.stream()
+                .filter(x -> x.hasConflictAnnotations)
+                .map(x -> "\n.put(\"" + getSMRFunctionName(x.method) + CONFLICT_METHODS_SUFFIX + "\", " +
```

Comment:
So, looking over this again, in #508 I introduce a ConflictFn, which if not specified should be any parameter annotated with @ConflictParameter. Shouldn't you just generate the lambda directly in the map, rather than generating an additional function? So - if it's a conflictFn, add a call to the function, and if not, just return the correct fields. The function should be of type IConflictFunction, with a signature of Object[] getConflictObjects(Object[] params).